### PR TITLE
Added search bar in navbar

### PR DESCRIPTION
**Pull request by al3xandra33**

### CODE REVIEW

File: app/models/offer.rb [hunk 1]
```diff
@@ -1,8 +1,20 @@
 class Offer < ApplicationRecord
   belongs_to :user
+
+  include PgSearch::Model
+  pg_search_scope :global_search,
+    against: [ :title, :description ],
```
Comment (anjali-kr) [inline app/models/offer.rb]: We should category as well for the search.

Comment (anjali-kr) [inline app/models/offer.rb]: We should category as well for the search.

File: app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -1,9 +1,16 @@
 <div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
   <div class="container-fluid">
-    <%= link_to offers_path, class: "navbar-brand" do %>
+    <%= link_to "#", class: "navbar-brand" do %>
```
Comment (anjali-kr) [inline app/views/shared/_navbar.html.erb]: Why remove the link to all offers?

Comment (anjali-kr) [inline app/views/shared/_navbar.html.erb]: Why remove the link to all offers?

File: db/seeds.rb [hunk 3]
```diff
@@ -18,6 +18,10 @@
 #   Movie.create(title: title, overview: overview, poster_url: poster_url, rating: rating)
 # end
 
+Offer.destroy_all
+User.destroy_all
+Booking.destroy_all
```
Comment (anjali-kr) [inline db/seeds.rb]: Bookings should be destroyed first, before users and offers. 
Deleting a user or offer linked to an existing booking will fail validation.

Comment (anjali-kr) [inline db/seeds.rb]: Bookings should be destroyed first, before users and offers. 
Deleting a user or offer linked to an existing booking will fail validation.

File: app/models/offer.rb (hunk 1 above)
Comment (al3xandra33) [reply app/models/offer.rb]: done

File: app/views/shared/_navbar.html.erb (hunk 2 above)
Comment (al3xandra33) [reply app/views/shared/_navbar.html.erb]: by mistake

File: db/seeds.rb (hunk 3 above)
Comment (al3xandra33) [reply db/seeds.rb]: this was part of trying to solve the pg_search gem problem...I commented out the 3 rows.

**Review by anjali-kr — APPROVED** [review]

**Pull request merged**